alica_engine/include/engine/AlicaContext.h

```diff
@@ -29,6 +29,7 @@ namespace alica
 
 class AlicaEngine;
 class IAlicaCommunication;
+class AlicaTestsEngineGetter;
```

Comment:
Having to forward declare this struct here & then depend on alica tests to define this struct seems like a strange way to do support this functionality. Can't we instead have an AlicaTestContext class that inhertis from TestContext & then define the engine getter there?

Reply:
It will only exist in the alica_tests package in future.